mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml

```diff
@@ -6,12 +6,12 @@ metadata:
   labels:
     app: busybox-sample
 spec:
-  preferredCluster: "akrai-s19-c1"
```

Comment:
We should not hardcode cluster name here

Reply:
done

Reply:
done